recipes/perl-sys-cpu/meta.yaml

```diff
@@ -0,0 +1,44 @@
+# This file created by conda-build 3.0.15+197.g41d59b5
+# meta.yaml template originally from:
+# /home/rdonnelly/conda/staged-recipes/recipes/perl-sys-cpu/0.61, last modified Sat Nov 11 23:55:48 2017
+# ------------------------------------------------
+#downgrade for perl 5.22 june 4th 2018
+{% set name = "perl-sys-cpu" %}
+{% set version = "0.61" %}
+{% set hash = "250a86b79c231001c4ae71d2f66428092a4fbb2070971acafd471aa49739c9e4" %}
+ 
+package:
+    name: {{ name }}
+    version: {{ version }}
+source:
+    fn: Sys-CPU-0.61.tar.gz
+    url: https://cpan.metacpan.org/authors/id/M/MZ/MZSANFORD/Sys-CPU-0.61.tar.gz
+    sha256: {{ hash }}
+build:
+    number: '0'
+    skip: true  # [win]  
+    skip: true  # [osx]
+requirements:
+    host:
+        - {{ compiler('c') }}
+
+        #- perl-threaded >=5.22.0
```

Comment:
Remove these commented out lines

Reply:
all the changes are done on perl-sys-cpu-feedstock